.devcontainer/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM mcr.microsoft.com/devcontainers/python:3.9
 
 RUN apt-get update
-RUN	apt-get --yes install rabbitmq-server
+RUN	apt-get --yes install libcairo2-dev rabbitmq-server
```

Comment:
Are you saying Python3.9 upgrade requires `libcairo2-dev`? 🤔  Do we know what the dependencies causes this?

Reply:
This is the error that I see while building the dev container when the package isn't installed:

```
Building wheels for collected packages: pycairo
  Building wheel for pycairo (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for pycairo (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [15 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-aarch64-cpython-39
      creating build/lib.linux-aarch64-cpython-39/cairo
      copying cairo/__init__.py -> build/lib.linux-aarch64-cpython-39/cairo
      copying cairo/__init__.pyi -> build/lib.linux-aarch64-cpython-39/cairo
      copying cairo/py.typed -> build/lib.linux-aarch64-cpython-39/cairo
      running build_ext
      Package cairo was not found in the pkg-config search path.
      Perhaps you should add the directory containing `cairo.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'cairo', required by 'virtual:world', not found
      Command '['pkg-config', '--print-errors', '--exists', 'cairo >= 1.15.10']' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pycairo
Failed to build pycairo
ERROR: Could not build wheels for pycairo, which is required to install pyproject.toml-based projects
```

One of the dependency bumps might be the first to use `pyproject.toml` instead of `setup.py` 🤔